### PR TITLE
I38 create manage institution page

### LINKED ISSUE #38: Create Manage Institution Page
## Basic Information
- Header - Use header component 
- Search bar - Use search bar component 
- List of institutions
- edit button for each existing institution
- A button to add Institution
- Add ability to filter for name
- Add dummy Institution json data

This page should be similar to the adjudicator page on branch i34. Feel free to change fonts/styling on header and search bar component :)

### PR DESCRIPTION
**Pull request by Ch3ngL0rd**
## Change Summary
Created the manage institution page - copied a lot off the "adjudicators page"

### Change Form
- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**
closes #38 

# Related issue

- Resolve #38

### CODE REVIEW

File: pages/manageinstitutions.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<script setup>
```
Comment (ChantelleKerr) [inline pages/manageinstitutions.vue:1]: change the file name to institutions.vue

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Just a file name change please :) 
Good work !

**Comment by Ch3ngL0rd** [comment]
Changed file name to 'institutions.vue'

**Review by ChantelleKerr — APPROVED** [review]
Good job :)

**Pull request merged**